### PR TITLE
Changing chainID

### PR DESCRIPTION
**Pull request by lg8yte**
This PR changes the chainID + fixes the unit tests

### CODE REVIEW

File: tests/testcases/manual.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
     "name": "completeTransferExpert",
     "tx": {
       "account_number": "0",
-      "chain_id": "cosmoshub-3",
+      "chain_id": "comsoshub-4",
```
Comment (jleni) [inline tests/testcases/manual.json]: There is a typo here

Comment (jleni) [reply tests/testcases/manual.json]: well.. almost everywhere :)

**Review by jleni — CHANGES_REQUESTED** [review]

File: tests/testcases/manual.json (hunk 1 above)
Comment (jleni) [reply tests/testcases/manual.json]: This makes me wonder how we didn't get it Zemu

Comment (lg8yte) [reply tests/testcases/manual.json]: Because the type was also in the app haha...fixed now!

**Comment by jleni** [comment]
One more thing I noticed..
Can we bump the version number?

**Pull request merged**